getpaid_test_project/getpaid_test_project/settings_test.py

```diff
@@ -38,25 +38,3 @@
     },
 
 }
-
```

Comment:
damn, I just remove it to use pdb in tests for my own purpose. Forgot to checkout that file before commit :(